test/functional/comment_controller_test.rb

```diff
@@ -235,6 +235,23 @@ def teardown
     assert_template 'comments/delete.js.erb'
   end
 
+  test 'should create a comment with status 4 if user has never created nothing before' do
+    user = users(:user_first_time_poster)
+    UserSession.create(user)
+
+    post :create, params: { id: nodes(:one).nid, body: 'example' }, xhr: true
+
+    comment = Comment.last
+    assert_equal user.id, comment.author.id
+    assert_equal 4, comment.status
+  end
+
```

Comment:
Don't you think these 2 tests can be combined? Also, you have to create one more functional test for the case when user is existing user and have created content, checking that moderator mail is not sent. Thanks!

Reply:
Oh, ok, I'm gonna still working on this. Please, ignore the submitted task hahaha